backend/src/schema/resolvers/searches.spec.js

```diff
@@ -0,0 +1,452 @@
+                      },
+                    ],
+                  },
+                })
+              })
+            })
+          })
+
+          describe('a post that contains a simple mathematical exprssion and linebreaks', () => {
+            describe('query a part of the mathematical expression', () => {
+              it('finds that post', async () => {
+                variables = { query: '(a - b)²' }
+                await expect(query({ query: searchQuery, variables })).resolves.toMatchObject({
+                  data: {
+                    findResources: [
+                      {
+                        __typename: 'Post',
+                        id: 'c-post',
+                        title: 'Die binomischen Formeln',
+                        content: `1. binomische Formel: (a + b)² = a² + 2ab + b²<br>
+2. binomische Formel: (a - b)² = a² - 2ab + b²<br>
+3. binomische Formel: (a + b)(a - b) = a² - b²`,
+                      },
+                    ],
+                  },
+                })
+              })
+            })
+
+            describe('query the same part of the mathematical expression without spaces', () => {
+              it('finds that post', async () => {
+                variables = { query: '(a-b)²' }
+                await expect(query({ query: searchQuery, variables })).resolves.toMatchObject({
+                  data: {
+                    findResources: [
+                      {
+                        __typename: 'Post',
+                        id: 'c-post',
+                        title: 'Die binomischen Formeln',
+                        content: `1. binomische Formel: (a + b)² = a² + 2ab + b²<br>
+2. binomische Formel: (a - b)² = a² - 2ab + b²<br>
+3. binomische Formel: (a + b)(a - b) = a² - b²`,
+                      },
+                    ],
+                  },
+                })
+              })
+            })
+
+            describe('query the mathematical expression over linebreak', () => {
+              it('finds that post', async () => {
+                variables = { query: '+ b² 2.' }
+                await expect(query({ query: searchQuery, variables })).resolves.toMatchObject({
+                  data: {
+                    findResources: [
+                      {
+                        __typename: 'Post',
+                        id: 'c-post',
+                        title: 'Die binomischen Formeln',
+                        content: `1. binomische Formel: (a + b)² = a² + 2ab + b²<br>
+2. binomische Formel: (a - b)² = a² - 2ab + b²<br>
+3. binomische Formel: (a + b)(a - b) = a² - b²`,
+                      },
+                    ],
+                  },
+                })
+              })
+            })
+          })
+
+          describe('a post that contains a poem', () => {
```

Comment:
I don't understand where the `poem` part makes a search different :thinking:

Reply:
It is just for the beauty of the poem. Be happy, that I did not include the second and third verse.